db/seeds.rb

```diff
@@ -3,5 +3,5 @@
 #
 # Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+admin = AdminUser.create!(firstname: "Muhammad", lastname: "Haroon", email: "admin@fintech.com", password: "Password123!@#")
```

Comment:
Use `find_or_create_by`. So if you make changes and ever need to run this file again. It won't rails model errors e.g. email uniqueness.